scripts/github-actions-packages

```diff
@@ -1,8 +1,14 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
+CRIU_REPO="https://download.opensuse.org/repositories/devel:/tools:/criu/xUbuntu_20.04"
```

Comment:
if I was to be pedantic this should be in the next commit. If you happen to do another rebase you could update, otherwise :shrug:

Reply:
Also correct and changed.